### PR TITLE
IA-1659: Switching 'no-value' from null to empty-string

### PR DESCRIPTION
**Pull request by bushong1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,9 +34,9 @@ Features:
 * `desired_capacity` - The desired number of containers running in the service. Default is `1`.
 * `max_capacity` - The maximum number of containers running in the service. Default is same as `desired_capacity`.
 * `min_capacity` - The minimum number of containers running in the service. Default is same as `desired_capacity`.
-* `scaling_metric` - A type of target scaling. Needs to be either `cpu` or `memory`. Default is `null`.
-* `scaling_threshold` - The percentage in which the scaling metric will trigger a scaling event. Default is `null`.
-* `efs_config` - The EFS id, root directory, and path. The module currently supports only one mount. Default is `null`.
+* `scaling_metric` - A type of target scaling. Needs to be either `cpu` or `memory`.
+* `scaling_threshold` - The percentage in which the scaling metric will trigger a scaling event.
+* `efs_config` - The EFS id, root directory, and path. The module currently supports only one mount.
```
Comment (elijahmuha) [inline README.md:39]: Add the default value into the description. Each variable has this even if it is an empty value.

File: README.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ Features:
 * `platform_version` - The ECS backend platform version; Defaults to `1.4.0` so EFS is supported.
 * `entrypoint_override` - Your Docker entrypoint command. Default is the `ENTRYPOINT` directive from the Docker image.
 * `command_override` - Your Docker command. Default is the `CMD` directive from the Docker image.
-* `task_policy_json` - A JSON formated IAM policy providing the running container with permissions. Default is `null`.
+* `task_policy_json` - A JSON formated IAM policy providing the running container with permissions.
```
Comment (elijahmuha) [inline README.md:53]: same here

File: README.md [hunk 3]
```diff
@@ -59,11 +59,11 @@ Features:
 * `public_subnet_ids` - A set of subnet ID's that will be associated with the Application Load-balancer. By default the module will use the default vpc's public subnets.
 * `security_group_ids` - A set of additional security group ID's that will associated to the Fargate service network interface. Default is `[]`.
 * `certificate_arn` - A certificate ARN being managed via ACM. If provided we will redirect 80 to 443 and serve on 443/https. Otherwise traffic will be served on 80/http.
-* `hosted_zone_id` - The hosted zone ID where the A record will be created. Required if `certificate_arn` and `service_fqdn` is set. Default is `null`.
+* `hosted_zone_id` - The hosted zone ID where the A record will be created. Required if `certificate_arn` and `service_fqdn` is set.
```
Comment (elijahmuha) [inline README.md:62]: Same

File: README.md [hunk 4]
```diff
@@ -59,11 +59,11 @@ Features:
...
 * `route53_allow_overwrite` - Set the `allow_overwrite` property of the route53 record.  If `true`, there will be no `terraform import` necessary for pre-existing records. Default is `false`.
-* `service_fqdn` - Fully qualified domain name (www.example.com) you wish to use for your service. Must be valid against the ACM cert provided. Required if `certificate_arn` and `hosted_zone_id` is set. Default is `null`.
-* `alb_log_bucket_name` - The S3 bucket name to store the ALB access logs in. Default is `null`.
-* `alb_log_prefix` - Prefix for each object created in ALB access log bucket. Default is `null`.
+* `service_fqdn` - Fully qualified domain name (www.example.com) you wish to use for your service. Must be valid against the ACM cert provided. Required if `certificate_arn` and `hosted_zone_id` is set.
+* `alb_log_bucket_name` - The S3 bucket name to store the ALB access logs in.
+* `alb_log_prefix` - Prefix for each object created in ALB access log bucket.
```
Comment (elijahmuha) [inline README.md:66]: Here too

**Review by elijahmuha — CHANGES_REQUESTED** [review]
Overall looks good, I just think we should keep the documentation consistent

File: README.md (hunk 1 above)
Comment (bushong1) [reply README.md:39]: I feel like if it's an optional variable and the default is "do nothing at all", then it doesn't really make sense to say what the "default value" is.

**Review by elijahmuha — APPROVED** [review]

**Pull request merged**